doc/classes/CylinderShape3D.xml

```diff
@@ -4,7 +4,7 @@
 		A 3D cylinder shape used for physics collision.
 	</brief_description>
 	<description>
-		A 2D capsule shape, intended for use in physics. Usually used to provide a shape for a [CollisionShape3D].
+		A 3D cylinder shape, intended for use in physics. Usually used to provide a shape for a [CollisionShape3D].
 		[b]Note:[/b] There are several known bugs with cylinder collision shapes. Using [CapsuleShape3D] or [BoxShape3D] instead is recommended.
 		[b]Performance:[/b] [CylinderShape3D] is fast to check collisions against, but it is slower than [CapsuleShape3D], [BoxShape3D], and [CylinderShape3D].
```

Comment:
Unsure what this was intended to mean, so didn't alter it, but assuming it should be:
```suggestion
		[b]Performance:[/b] [CylinderShape3D] is fast to check collisions against, but it is slower than [CapsuleShape3D], [BoxShape3D], and [SphereShape3D].
```

Reply:
Yeah, seems like a typo. Suggested change seems correct (from these sphere is the simplest one to check against). And other docs agree with the change:
https://github.com/godotengine/godot/blob/6588a4a29af1621086feac0117d5d4d37af957fd/doc/classes/Shape3D.xml#L8
https://github.com/godotengine/godot/blob/6588a4a29af1621086feac0117d5d4d37af957fd/doc/classes/CapsuleShape3D.xml#L8

Reply:
Will update soon

Reply:
Done!